src/pages/blog/sigcse2024_AffiliatedEvent/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import classnames from "classnames";
+import Head from "next/head";
+
+import { PageBanner } from "../../../components/Banner";
+import { Heading } from "../../../components/Heading";
+import Stack from "../../../components/Stack";
+
+export default function SIGCSE2024() {
+  return (
+    <React.Fragment>
+      <Head>
+        <title>SIGCSE 2024</title>
+      </Head>
+
+      <PageBanner
+        title="SIGCSE 2024"
+        subtitle="Technical Symposium on Computer Science Education"
+      />
+
+      <div className={classnames("container-fluid py-4")}>
+        <div className="container-md pt-2">
+          <Stack spacing={4}>
+            <Heading>
+              Affiliated Event: Spiffy Assessments with PrairieLearn
+            </Heading>
+
+            <p>
+              A key element in both mastery learning and competency-based
+              education involves providing students with multiple opportunities
+              to practice with rapid feedback, a feature provided by
+              autograders. PrairieLearn is an open-source assessment authoring
+              system that empowers the development of exceptionally rich
+              interactive STAR questions &#8212; Skill-specific, Tagged to
+              course competencies, Auto-graded, and Randomizable.
```

Comment:
This is a new term for me - is this a Berkeley thing? Are we going to make this part of our marketing?

Reply:
I do not think we are using this for our marketing. This is a Berkeley thing, and what we included in the proposal. Do you see an issue using that term here?

Reply:
Only the fact that it's Berkeley-specific. But I guess that's fine given that they're cohosts of the event.